saleor/product/filters.py

```diff
@@ -16,7 +17,7 @@
                       'Product list sorting option', 'price')}
 
 
-class ProductFilter(FilterSet):
```

Comment:
Do you still need `FilterSet` in that file then?

Reply:
No